<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>William Findlay</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ebpH</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/assets/img/logos/favicon.png</normaloff>:/img/assets/img/logos/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actiontest"/>
    <addaction name="action_Force_Save_Profiles"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionebpH_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Actions">
    <property name="title">
     <string>&amp;Monitoring</string>
    </property>
    <addaction name="action_Start_Monitoring"/>
    <addaction name="action_Stop_Monitoring"/>
    <addaction name="separator"/>
    <addaction name="action_Inspect_Executable"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Actions"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About ebpH</string>
   </property>
  </action>
  <action name="actionebpH_Help">
   <property name="text">
    <string>ebpH &amp;Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>&amp;Export Statistics</string>
   </property>
  </action>
  <action name="action_Start_Monitoring">
   <property name="text">
    <string>&amp;Start Monitoring</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Stop_Monitoring">
   <property name="text">
    <string>&amp;Stop Monitoring</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Inspect_Executable">
   <property name="text">
    <string>View/Modify &amp;Profile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Force_Save_Profiles">
   <property name="text">
    <string>Force &amp;Save Profiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Forget_Profile">
   <property name="text">
    <string>&amp;Forget A Profile</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
